instance/models/openedx_appserver.py

```diff
@@ -66,6 +66,11 @@ class Meta:
         'The default contact email for this instance; also used as the from address for emails '
         'sent by the server.'
     ))
+    privacy_policy_url = models.URLField(
```

Comment:
Add a fields for TOS too.

Reply:
Is ToS required here as well? The ticket and discovery exclusively mention Privacy Policy, whereas BB-985 explicitly mentions ToS in addition to Privacy Policy.